src/commons/error.rs

```diff
@@ -649,6 +657,9 @@ impl Error {
 
             // internal server error
             Error::AggregateStoreError(e) => ErrorResponse::new("sys-store", &self).with_cause(e),
+            
+            // internal server error
+            Error::WalStoreError(e) => ErrorResponse::new("sys-wal-store", &self).with_cause(e),
```

Comment:
You distinguish wal store error from aggregate store error here, but above you use the same description string for both. This seems inconsistent.

Reply:
updated the error text to be a bit more clear (none of this is likely to be seen by users, but still)